tools/downloader/list_topologies.yml

```diff
@@ -1533,6 +1590,7 @@ topologies:
       Head pose estimation network based on simple, handmade CNN architecture. Angle
       regression layers are convolutions + ReLU + batch norm + fully connected with
       one output.
+    type: "headpose"
```

Comment:
I think this ought to be called `head_pose_estimation` for consistency with others (especially `human_pose_estimation`).

Reply:
done